doc/diffusion.bib

```diff
@@ -319,14 +319,14 @@ @Article{Carlsson2009
 }
 
 @Article{Avram2008NMRBiomed,
-  Author         = {Avram, Liat and \�{O} zarslan, Evren and Assaf, Yaniv
+  Author         = {Avram, Liat and \�{O} zarslan, Evren and Assaf, Yaniv
```

Comment:
This symbol and other similar ones introduced are not plain, BIbTeX symbols; e.g. the correct BibTeX spelling for Evren's surname would be `\"{O}zarslan`. I can have a go to the file to fix them. I assume the changes were done inadvertently introduced when running some tool.

Reply:
I can have a look at this and amend the commit.

Reply:
Removed existing entries were removed in PR #3308. Resolving this.